automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/PublishSpec.scala

```diff
@@ -117,4 +140,17 @@ class PublishSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Wo
       }
     }
   }
+
+  "DUOS autocomplete" - {
+    "should give multiple results for a partial word" in withWebDriver { implicit driver =>
+      val user = UserPool.chooseAnyUser
+      // there is no need for an auth token for this test, except that the api wrapper expects one
+      implicit val authToken: AuthToken = user.makeAuthToken()
+      val partialTextQuery = "dis"
```

Comment:
This is relying on externally populated data. I'd prefer a more self-contained test here but I understand that that's difficult in this case. Can there be a comment about what some of the expected results should be and where they're coming from (as in where to look for the data in the fiab-populate code)?